src_py/__init__.py

```diff
@@ -222,24 +228,37 @@ def warn_unwanted_files():
 try:
     from pygame.surface import Surface, SurfaceType
 except (ImportError, IOError):
-    Surface = lambda: Missing_Function
+
+    def Surface(size, flags, depth, masks):  # pylint: disable=unused-argument
+        _attribute_undefined("pygame.Surface")
+
+    SurfaceType = Surface
 
 try:
     import pygame.mask
     from pygame.mask import Mask
 except (ImportError, IOError):
     mask = MissingModule("mask", urgent=0)
-    Mask = lambda: Missing_Function
+
+    def Mask(size, fill):  # pylint: disable=unused-argument
+        _attribute_undefined("pygame.Mask")
+
 
 try:
     from pygame.pixelarray import PixelArray
 except (ImportError, IOError):
-    PixelArray = lambda: Missing_Function
+
+    def PixelArray(surface):  # pylint: disable=unused-argument
+        _attribute_undefined("pygame.PixelArray")
+
 
 try:
     from pygame.overlay import Overlay
 except (ImportError, IOError):
-    Overlay = lambda: Missing_Function
+
+    def Overlay(format, size):  # pylint: disable=unused-argument
```

Comment:
This function is just a shim that errors more cleanly ATM, IMHO we can keep this around because we have things like `MissingModule` anyways, so this is a `Missing_Function` implementation that errors better